antissh.py

```diff
@@ -27,7 +30,11 @@
 NICKNAME = config.get('host', 'nickname', fallback='antissh')
 SERVER_PASSWORD = config.get('host', 'password', fallback=None)
 MODES = config.get('host', 'modes', fallback='')
-KLINE_CMD_TEMPLATE = config.get('host', 'kline_cmd', fallback='KLINE 86400 *@{ip} :Vulnerable SSH daemon found on this host.  Please fix your SSH daemon and try again later.\r\n')
+KLINE_CMD_TEMPLATE = config.get(
+    'host', 'kline_cmd',
+    fallback='KLINE 86400 *@{ip} :Vulnerable SSH daemon found on this host.  '
+             'Please fix your SSH daemon and try again later.\r\n',
```

Comment:
Is it intended for this to be in two parts now?

Reply:
2 string literals one after the other just get concatenated, so it's the same string, just not as a hugely-long line.